config/samples/topolvm_v1_logicalvolume.yaml

```diff
@@ -4,6 +4,4 @@ metadata:
   name: logicalvolume-sample
 spec:
   # Add fields here
-  size: 10Gi
-  node_name: localhost
-  name: pvc-1234
+  foo: bar
```

Comment:
Is this correct?

Reply:
It is an automatically generated sample file.
Removing.